mongo/client.go

```diff
@@ -955,6 +955,11 @@ func (c *Client) BulkWrite(ctx context.Context, writes []ClientBulkWrite,
 			op.rawData = &rawData
 		}
 	}
+	if additionalCmd := optionsutil.Value(bwo.Internal, "addCommandFields"); additionalCmd != nil {
+		if ac, ok := additionalCmd.(bson.D); ok {
+			op.additionalCmd = ac
+		}
+	}
```

Comment:
Combine these conditions into one. This suggestion applies to all instances of this block in this PR.


```suggestion
	if ac, ok := optionsutil.Value(bwo.Internal, "addCommandFields").(bson.D); ok {
			op.additionalCmd = ac
	}
```